### DISCUSSION TITLE
When logging in, XRDP blue screens

### DISCUSSION THREAD

**Discussion by Aerospace911** [neutrinolabs/xrdp#2751] [Q&A]
![image](https://github.com/neutrinolabs/xrdp/assets/79132854/95a5091d-0d01-4ae4-803d-35c71ccb9429)

please let me know what logs i need to attach. I built the latest version of xrdp using git clone in the documentation

**Comment by Aerospace911** [comment]
I now cannot login to my linux computer (key board and mouse dont work) however the xrdp is started when i power on the computer but i cant login there either

**Reply by Aerospace911** [reply]
Update: I reinstalled xrdp from the recovery mode and now can access the computer remotely. But the The keyboard and mouse still dont work. IE. when the computer goes to sleep, keyboard and mouse does not wake the computer up, or when I boot the computer, the mouse is in the center of the screen and the mouse and keyboard dont work. However, when I am in recovery mode the keyboard works.

**Reply by XR-stb** [reply]
you should log out you current user, and in destination virtual machine install xrdp service and start it.

here is my question. you can be no care.
And more bum, i ubuntu has three user: xiaoran, root, test1, i can use xrdp to connect root and test1, but when i connect xiaoran, the window will crash, i don know how to see the log, if someone can tell me, that's very thanks
